packages/istanbul-lib-report/README.md

```diff
@@ -3,4 +3,28 @@
 [![Greenkeeper badge](https://badges.greenkeeper.io/istanbuljs/istanbul-lib-report.svg)](https://greenkeeper.io/)
 [![Build Status](https://travis-ci.org/istanbuljs/istanbul-lib-report.svg?branch=master)](https://travis-ci.org/istanbuljs/istanbul-lib-report)
 
-Core reporting utilities for istanbul. Docs to follow.
+Core reporting utilities for istanbul.
+
+## Example usage
+
+```js
+const libReport = require('istanbul-lib-report');
+const reports = require('istanbul-reports');
```

Comment:
Could we destructure these imports? That makes reading examples like this [from other docs] clearer for me.

Reply:
since the used variables are "create" and "createContext" and therefore to be clear the imports would need to be renamed, I feel like destructuring in this case makes the code less readable.